core/templates/pages/topics-and-skills-dashboard-page/skills-list/skills-list.component.ts

```diff
@@ -51,17 +51,17 @@ interface MergeModalResult {
   templateUrl: './skills-list.component.html'
 })
 export class SkillsListComponent {
-  @Input() skillSummaries: AugmentedSkillSummary[];
-  @Input() pageNumber: number;
-  @Input() itemsPerPage: number;
-  @Input() editableTopicSummaries: CreatorTopicSummary[];
-  @Input() mergeableSkillSummaries: SkillSummary[];
-  @Input() untriagedSkillSummaries: SkillSummary[];
-  @Input() userCanDeleteSkill: boolean;
-  @Input() userCanCreateSkill: boolean;
-  @Input() skillsCategorizedByTopics: SkillsCategorizedByTopics;
+  @Input() skillSummaries!: AugmentedSkillSummary[];
+  @Input() pageNumber!: number;
+  @Input() itemsPerPage!: number;
+  @Input() editableTopicSummaries!: CreatorTopicSummary[];
+  @Input() mergeableSkillSummaries!: SkillSummary[];
+  @Input() untriagedSkillSummaries!: SkillSummary[];
+  @Input() userCanDeleteSkill: boolean = false;
+  @Input() userCanCreateSkill: boolean = false;
```

Comment:
Why are these `false` by default?

Reply:
Pardon me, I didn't pay attention to this detail. Will correct it.

Reply:
But I was genuinely asking :), I'm not sure what should be the correct value, you will need to analyze the code a provide an explanation why you think it should be either `true` or `false`.

Reply:
I tried investigating the use of the two properties.  Neither of them is further assigned any value in the file and `true` seems to be the natural assignment to whether a user can create or delete a skill. Thus, in my opinion, both should be `true`.